<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input</class>
 <widget class="QDialog" name="Input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Stream</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Stream Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="radiobuttomHot">
     <property name="text">
      <string>Hot</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="radioButtonCold">
     <property name="text">
      <string>Cold</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTs">
     <property name="text">
      <string>Supply Temperature</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditTs"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="labelTs_2">
     <property name="text">
      <string>(°C)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelTt">
     <property name="text">
      <string>Target Temprature</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditTt"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelTt_2">
     <property name="text">
      <string>(°C)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelHCF">
     <property name="text">
      <string>Heat Capacity Flowrate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditHCF"/>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="labelHCF_2">
     <property name="text">
      <string>(kW/°C)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radiobuttomHot</tabstop>
  <tabstop>radioButtonCold</tabstop>
  <tabstop>lineEditTs</tabstop>
  <tabstop>lineEditTt</tabstop>
  <tabstop>lineEditHCF</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Input</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
